partials/time_to_retirement.html

```diff
@@ -34,102 +34,105 @@
     </div>
     <div class="container-fluid output-controls-section">
         <div class="row">
-            <div class="graph-section supporting-content">
+            <div class="graph-section supporting-content col-xs-12">
                 <div class="supporting-content-wrapper">
                     <div class="col-md-8" id="graph-wrapper">
                         <div id="retirement-graph"></div>
                         <div id="explain-text">Financial independence is based on <a href="#/about">the 4% Safe Withdrawal Rate</a></div>
                     </div>
-                    <div class="supporting-section-body col-md-4">
-                        <div id="expenses-control" class="input-control col-md-offset-3 col-md-5">
-                            <div class="">
-                            	<div class="">
-						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="expenses = expenses + 100"></button>
+
+                    <div class="output-control-container col-xs-12 col-md-4">
+                        <div id="expenses-control" class="input-control primary-control col-xs-offset-3 col-xs-6 col-md-offset-3 col-md-6">
+                            <div>
+                            	<div>
+						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="incrementOutputValue('expenses', 100)"></button>

```

Comment:
Ah yes, this is one of the more annoying aspects of angular. For some reason, passing literals (just a number, or string, basically non-objects) breaks the scope connection when passed around like that. I'm not sure why, but there's info online if you're interested. The solution is to put all the values in an object in the scope. What I mean is instead of using `$scope.scope_variable`, use `$scope.scope_object.value`. Then you can pass the whole object, `$scope.scope_object` and increment `$scope.scope_object.value` in the controller and the binding shouldn't be broken.

If that's too confusing or gives you more problems then don't worry about it and just leave it as it was, not a huge deal.